lib/src/over_react_test/js_component.dart

```diff
@@ -13,12 +13,12 @@
 // limitations under the License.
 
 import 'package:js/js.dart';
-import 'package:react/react.dart' as react;
+import 'package:react/react.dart' as react show div;
```

Comment:
not sure tbh @jacehensley-wf  - is it a problem? I may have just done it to make it easier to grok what specifically is needed from `react.dart` in this file since in general - we want to minimize direct dependencies on react, and prefer depending on the exports provided in `over_react`.